cli/src/config/merge_tools.toml

```diff
@@ -52,10 +52,13 @@ merge-args = ["--wait", "--merge", "$left", "$right", "$base", "$output"]
 # markers. Unfortunately, it does not seem to be able to output conflict markers when
 # the user only resolves some of the conflicts.
 merge-tool-edits-conflict-markers = true
+# VS Code prefers Git-style conflict markers
+conflict-marker-style = "git-diff3"
```

Comment:
Set "snapshot" for vimdiff? I don't use vimdiff but I'm pretty sure I've heard that vimdiff users would prefer that. Maybe some vimdiff user can confirm?

Reply:
Per @yuja's suggestion, I'll be moving the merge tool changes to a separate PR: #4951.